src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-color-picker {
+  width: 250px;
+  border-radius: 6px;
+  box-shadow: $iui-elevation-2;
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 10px;
+  padding-top: 10px;
+  @include themed {  background: rgb(255, 255, 255);}
```

Comment:
What were you trying to achieve here? This looks like you started something and abandoned it.

Reply:
It was to make the background white, but I think it is not necessary

Reply:
You should be using the `iui-color-background-1` variable for that, as now it's a white background even on dark theme.

But if you use the `iui-dropdown` mixin, you won't need it I think